components/raftstore-v2/src/batch/store.rs

```diff
@@ -97,6 +96,11 @@ pub struct StoreContext<EK: KvEngine, ER: RaftEngine, T> {
     /// A background pool used for high-priority works.
     pub high_priority_pool: FuturePool,
 
+    // current_time and node_start_time relate to leader lease,
+    // must use monotonic_raw_now.
+    pub current_time: Option<Timespec>,
+    pub node_start_time: Option<Timespec>,
```

Comment:
how about using `unsafe_vote_deadline` and calculate it by start time at start. A optional start time is confusing.

Reply:
By taking timespec, it can skip comparing timespec and saves accessing monotic_raw_now clock (it's a syscall).